File: Microservice/5/customer-service/src/main/java/com/example/demo/model/Customer.java
package com.example.demo.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;

@Entity
@Data
@Table(name = "customer_table")
public class Customer {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	@Column(name = "customer_id")
	private String customerId;
	@Column(name = "first_name")
	private String firstName;
	@Column(name = "last_name")
	private String lastName;
	private String email;
}

File: typescript_projects/customer.js
var Customer = /** @class */ (function () {
    function Customer() {
        //properties
        this.firstName = 'Marry';
        this.lastName = 'Public';
    }
    return Customer;
}());
//object of customer class
var myCustomer = new Customer();
console.log(myCustomer.firstName);
console.log(myCustomer.lastName);
